packages/vitest/globals.d.ts

```diff
@@ -11,5 +11,6 @@ declare global {
   const afterAll: typeof import('vitest')['afterAll']
   const beforeEach: typeof import('vitest')['beforeEach']
   const afterEach: typeof import('vitest')['afterEach']
+  let __vitest_worker__: import('vitest').WorkerGlobalState
```

Comment:
We shouldn't expose this to user type. Let's have it in the source code which will be removed in build